packages/components/src/components/prompts/ConfirmOnDevice/index.tsx

```diff
@@ -99,10 +112,12 @@ const isStepActive = (index: number, activeStep?: number) => {
 
 interface Props {
     title: React.ReactNode;
-    successText: React.ReactNode;
+    successText?: React.ReactNode;
```

Comment:
If the success text can be optional, it would probably be best to handle it in the render to prevent an empty success message (see L142).